src/marker.rs

```diff
@@ -1290,4 +1323,12 @@ mod test {
     fn test_closing_parentheses() {
         MarkerTree::from_str(r#"( "linux" in sys_platform) and extra == 'all'"#).unwrap();
     }
+
+    #[test]
+    fn test_evaluate_extras() {
+        let marker_tree =
+            MarkerTree::from_str(r#"("linux" in sys_platform) and extra == 'day'"#).unwrap();
+        assert!(marker_tree.evaluate_extras(&["day"]));
```

Comment:
So is the question here effectively: "Could this marker _ever_ evaluate to `true` given the current set of `extras`?"

Reply:
yes, good point, updated the doc strings